python/data_extraction/export_data.py

```diff
@@ -1,63 +1,78 @@
 from data_extraction import content_export
 from data_extraction import taxonomy_query, plek
-import json
-import sys
+from data_extraction import json_arrays
+from data_extraction.helpers import dig
 import functools
 from multiprocessing import Pool
+import gzip
+import json
+import os
+
 
+config_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', 'config', 'data_export_fields.json')
 
-with open('config/data_export_fields.json') as json_data_file:
+with open(config_path) as json_data_file:
     configuration = json.load(json_data_file)
 
 
-content_links_generator = content_export.content_links_generator(
-    blacklist_document_types=configuration['blacklist_document_types']
-)
+def __transform_content(input_filename="data/content.json.gz",
+                        output_filename="data/filtered_content.json.gz",
+                        transform_function=lambda x: x):
+    with gzip.open(input_filename, mode='rt') as input_file:
+        with gzip.open(output_filename, mode='wt') as output_file:
+            content_generator = json_arrays.read_json(input_file)
+            json_arrays.write_json(output_file, transform_function(content_generator))
 
 
-slicer = functools.partial(content_export.content_hash_slicer,
-                           base_fields=configuration['base_fields'],
-                           taxon_fields=configuration['taxon_fields'],
-                           ppo_fields=configuration['ppo_fields'])
+def export_content(output_filename="data/content.json"):
```

Comment:
It looks like this commit is changing the format to be gzipped always. If so, the default filename should probably be `"data/content.json.gz"`.

Reply:
fixed